forms-flow-api/src/formsflow_api/services/form_embed.py

```diff
@@ -65,7 +65,9 @@ def application_create_with_submission(data, formio_url, web_url, token):
             raise BusinessException(response, status) from application_err
 
     @staticmethod
-    def __populate_default_keys(form_data: Dict) -> Dict:
+    def __populate_default_keys(  # pylint: disable=unused-private-member
```

Comment:
if the private method is unused you can remove the method. or try placing public members after private members